pyvisfile/vtk/vtk_ordering.py

```diff
@@ -210,7 +209,7 @@ def vtk_lagrange_simplex_node_tuples(dims, order, is_consistent=False):
 
 
 def vtk_lagrange_simplex_node_tuples_to_permutation(node_tuples):
-    order = max([sum(i) for i in node_tuples])
+    order = max([max(i) for i in node_tuples])
```

Comment:
I agree that these two are equivalent, but I'm still curious why this change happened, especially given that the function name mentions "simplex".

Reply:
Mostly just to have it consistent with the quad one below. I initially copy pasted it and it obviously didn't work there.